docs/sources/developer/Developing-Windows.md

```diff
@@ -1,55 +1,68 @@
 # Developing on Windows
 
-To download the latest version of Plaso you'll need to install the git tools: http://git-scm.com/downloads
+## Git
```

Comment:
Add text about the preferred python version? 3.7 ?

Reply:
I'd like to overhaul all these instructions, but I'll do that in a later change.

Reply:
ack